.changes/next-release/bugfix-EndpointProviderStandardLibrary-21046.json

```
@@ -0,0 +1,5 @@
+{
+  "type": "enhancement",
+  "category": "Endpoint Provider Standard Library",
+  "description": "Correct spelling of 'library' in `StandardLibrary` class"
```

```suggestion
  "description": "Correct spelling of 'library' in ``StandardLibrary`` class"
```

Our changelog is reStructuredText, not Markdown, so inline literal text must have two backticks. This mistake has crept in a few times before, for example in 1.29.0.